types/events.go

```diff
@@ -39,8 +43,8 @@ func NewEventManager() *EventManager {
 	return NewEventManagerWithHistory([]abci.Event{})
 }
 
-// GetEventHistory returns a deep copy of the ABCI events that have been
-// committed to thus far.
+// GetEventHistory returns a deep copy of the ABCI events history.
+// This should only be called in EndBlock processing.
 func (em *EventManager) GetABCIEventHistory() []abci.Event {
```

Comment:
Oh good, an opportunity to fix this typo!
```suggestion
// GetABCIEventHistory returns a deep copy of the ABCI events history.
// [AGORIC] This should only be called in EndBlock processing.
func (em *EventManager) GetABCIEventHistory() []abci.Event {
```

Reply:
Done!